java/test/jmri/jmrix/loconet/LnThrottleManagerTest.java

```diff
@@ -642,25 +644,18 @@ public void notifyDecisionRequired(LocoAddress address, DecisionType question) {
         Assert.assertEquals("No loconet traffic at throttle2 release", 0, lnis.outbound.size());
 
         Assert.assertTrue("Address still required",
-                InstanceManager.throttleManagerInstance().addressStillRequired(260));
+                tm.addressStillRequired(260));
 
         tm.releaseThrottle(throttle, throtListen);
         Assert.assertFalse("Address no longer required",
-                InstanceManager.throttleManagerInstance().addressStillRequired(new DccLocoAddress(260, true)));
-        // Wait for 2 messages, the set slot common and then the get slot refresh
-        JUnitUtil.waitFor(() -> {
-            return !lnis.outbound.isEmpty();
-        },"wait for message");
-        JUnitUtil.waitFor(() -> {
-            return !lnis.outbound.elementAt(lnis.outbound.size() -1).toString().equals("B5 09 10 00");
-        }, "COMMON");
-        JUnitUtil.waitFor(() -> {
-            return !lnis.outbound.elementAt(lnis.outbound.size() -1).toString().equals("BB 09 00 00");
-        }, "Slot read");
-
+                tm.addressStillRequired(new DccLocoAddress(260, true)));
+        
+        JUnitUtil.waitFor(()->{return !lnis.outbound.isEmpty(); },"didn't get the LocoNet message");
+        Assert.assertEquals("[B5 09 10 00]", lnis.outbound.toString());
```

Comment:
It would be good if you here expect both the `COMMON` and the `Slot read` message. Which means that you will have to wait for the message queue to have two messages.

It would also be good to add comments about what the messages are:
```
// B5 09 10 00 = COMMON
// BB 09 00 00 = Slot read
```